### DISCUSSION TITLE
Welcome to AMDMIGraphX Discussions!

### DISCUSSION THREAD

**Discussion by causten** [ROCm/AMDMIGraphX#2044] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qianqing13579** [comment]
Hello @causten ,I am a university student. In our lab,we use AMD GPU to do experiments and projects.I use MIGraphX to do inference,and I am very interested in it. We have many projects which need dynamic shape,and I have been developing dynamic shape of migraphx recently

**Reply by causten** [reply]
Yes fantastic, Keep working with Charlie.  I'd love to hear more about your lab and migraphx usage
